service-map/src/main/java/fi/nls/oskari/map/layer/formatters/LayerJSONFormatterUSERLAYER.java

```diff
@@ -54,4 +62,18 @@ private static String getUserLayerTileUrl() {
         }
         return PROPERTY_RENDERING_URL + "&id=";
     }
+
+    // creates JSONArray from JSONArray's JSONObject keys [{key,value},{key2,value2},..] -> [key1, key2,..]
+    private static JSONArray createJSONArrayJSONObjectKeys(final JSONArray json) {
+        try {
+            JSONArray jsarray =  new JSONArray();
+            for(int i = 0; i < json.length(); ++i){
+                JSONObject obj = json.getJSONObject(i);
+                jsarray.put(obj.names().get(0));
```

Comment:
To get the first key could call JSONObject#keys() and get the first value from the Iterator (should probably check that atleast one exists)